cli/cli.js

```diff
@@ -46,7 +48,14 @@ program
             .default(defaults.FORMAT),
     )
     .showHelpAfterError()
-    .action(run);
+    .action(run)
```

Comment:
We should probably rename this function:
```suggestion
    .action(list)
```

Reply:
Done in 1db9c2dcb95f0e96916f9d15e489cb7906e61446.